examples/todo-backend/todo-backend-s2i.yaml

```diff
@@ -2,12 +2,14 @@
 # quickstart on OpenShift with the Helm Chart for WildFly.
 build:
   uri: https://github.com/wildfly/quickstart.git
-  ref: 26.0.0.Final
+  ref: 27.0.0.Final
   contextDir: todo-backend
   mode: s2i
   s2i:
     featurePacks:
-      - org.wildfly:wildfly-galleon-pack:26.1.1.Final
+      - org.wildfly:wildfly-galleon-pack:27.0.0.Final
+      - org.wildfly:wildfly-datasources-galleon-pack:2.0.2.Final
```

Comment:
Should be version 3.0.0.Final. Generally we order wildfly/cloud/datasources.

Reply:
fixed, thanks